webapp/portlet/src/main/webapp/vue-apps/who-is-online-app/components/ExoWhoIsOnline.vue

```diff
@@ -36,7 +36,17 @@ export default {
       return this.users && this.users.length;
     },
   },
+  watch: {
+    display() {
+      if (this.display) {
+        this.$el.closest('.PORTLET-FRAGMENT').classList.remove('hidden');
+      } else {
+        this.$el.closest('.PORTLET-FRAGMENT').classList.add('hidden');
+      }
+    }
+  },
   mounted() {
+    this.$el.closest('.PORTLET-FRAGMENT').classList.add('hidden');
```

Comment:
Why should we add this ?
If needed to not display the Who's online Widget until getting the list of users, then you can test this in `main.js` since you have there the `onlineUsers.length` information to know if the widget will be display on page rendering or not, like you did for Suggestions Widget.

```suggestion
```